experimental/examples/bubblebench/editable-shared-tree/src/schema.ts

```diff
@@ -0,0 +1,234 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+    brand,
+    EditableTree,
+    emptyField,
+    EmptyKey,
+    FieldKinds,
+    fieldSchema,
+    JsonableTree,
+    namedTreeSchema,
+    rootFieldKey,
+    SchemaData,
+    ValueSchema,
+} from "@fluid-internal/tree";
+
+export const stringSchema = namedTreeSchema({
+    name: brand("String"),
+    extraLocalFields: emptyField,
```

Comment:
A while ago, I refactoerd our schema helpers which made it possible to make the `extraLocalFields` option default to `emptyField`, so you can remove that from all the schema in this file now.

Reply:
Done in latest commit.